global.json

```diff
@@ -1,5 +1,6 @@
 {
     "sdk": {
-      "version": "6.0.100-rc.2.21505.57"
+      "version": "6.0.100-rc.2.21505.57",
```

Comment:
RC2 isn't supported anymore, so might as well change this to one of the stable .NET 6 SDKs.

Reply:
Yes, I didn't want to change anything else, so I let it as-is.